back/src/routes/airtable.ts

```diff
@@ -91,43 +89,73 @@ airtableRouter.post("/getInfo", function (req, res) {
       contactMethod: "",
       paymentMethod: "",
     };
-    console.log("what is the req.body: " + JSON.stringify(req.body))
     base("Authentication")
       .select({ filterByFormula: `{Username} = "${userName}"` })
       .firstPage((err, records) => {
         if (err) console.error(err);
         if (records.length != 1)
           res.status(401).send({ error: "No such user exists" });
         const recordID = records[0].fields["User Data Record ID"];
```

Comment:
Nit: Make constants all caps

Reply:
It’s fine as != as it checks in both directions. Any more than 1 and a problem exists. Any less than one, a problem exists. As Mack stated though, it’s how they had checked originally.